public/components/layer_control_panel/layer_control_panel.scss

```diff
@@ -22,6 +22,12 @@
   .euiListGroupItem__label {
     width: $euiSizeL * 6;
   }
+
+  .euiDroppable {
+    overflow-y: auto;
+    overflow-x: hidden;
+    max-height: $euiSizeL * 8;
```

Comment:
What is the default value of this elastic ui variable $euiSizeL?

Reply:
Per euiSizeL is 24 px, so here max-height is 192 px. Per layer is 32 px height, maximum in screen displayed  layers number is 6.